### ISSUE TITLE
Ammo/Inventory Quantity Cumbersome to manage

### ISSUE THREAD

**Issue by xanstin** [vttred/ose#264]
### New Feature or Changed Behaviour

Currently Ammunition goes under misc at the bottom of the sheet. It cant be linked to weapons and doesn't auto deduct. if you have a large inventory and close the drop downs, adjusting the ammo count manually opens them all back up again each time. This makes tracking ammo extremely cumbersome.

Enhancement would be to be able to link ammo bundles to a weapon similar to other rule sets where it auto deducts. Another option is create a ammo category and place below weapons category at the top. 

Inventory itself is a bit cumbersome as you cant drag and drop from items directory into a container it has to go to misc first then dragged into a bag.

### Why?

This will reduce workflow and make tracking ammo easier each shot fired. Currently its a lot of up and down on the sheet and if you have a lot of closed tabs / bags to streamline inventory you have to redo this every time you adjust count on ammo or any consumable with a quantity.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by wyrmisis** [comment]
> Enhancement would be to be able to link ammo bundles to a weapon similar to other rule sets where it auto deducts. Another option is create a ammo category and place below weapons category at the top.

We've talked about auto-tracking ammo in the Dev channel on Discord. We've been doing a lot of behind the scenes work on item and actor data structure, in fact, to make stuff like this a little easier to implement.

I'd love to tackle this (in fact, it could likely happen as an extension of my work in #263 ), as long as we have some ideas for how to display this in the UI.

What I'm thinking:
* Use tags on ammunition items and missile weapons to relate them
  * This'll keep things flexible for folks who are using the Carcass Crawler ranged weapons (or other customizable weapons)
* Ammunition without a linked weapon appears under Misc.
* Ammunition with a linked weapon appears under the linked weapons (as in, multiple bows will track the same bundles of arrows). We'll use the same tick-mark UI that we use for ammo and other consumables right now.

I think it'd be useful to expand this to thrown weapons, too.

> Inventory itself is a bit cumbersome as you cant drag and drop from items directory into a container it has to go to misc first then dragged into a bag.

Would you mind capturing this in a separate issue, please? This is something I noticed as well as I've been starting to use the system more, and I'd like to see it fixed as well.

**Comment by xanstin** [comment]
Yeah those look like great options. I created a new item for the drag and dropping of inventory items.
